### DISCUSSION TITLE
Receive error message if sending email fails

### DISCUSSION THREAD

**Discussion by LickABrick** [pocketbase/pocketbase#3224] [Q&A] [answered]
Am I correct if I say that I do get 200 response when I call `requestVerification` even if the mail server would be unavailable or somehow PB fails to send an email?

I would like to inform a user on sign-up if the verification mail fails to send. Will probably not happen very often, but just in case.

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, we don't await the verification and password reset mails and this is intentional in order to avoid to some extend users enumerations. But even if we await the email send command, this will just mean that the mail was submitted to the mail server but that doesn't necessary guarantee that it will be delivered to the user.

Also note that there is a very rudimentary "flood protection" that will prevent sending the same verification email if the previous one was submitted < 2min.

**Reply by LickABrick** [reply]
Gotcha, thanks for the quick response. Hadn't thought of that. In case this does happen I guess we would inform the newly created users when the mail server is back online.
